### PR TITLE
The building docs process will continue if JSDoc fails due to errors

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: The building docs process will continue if JSDoc fails due to errors during the validation. See ckeditor/ckeditor5-dev#550.

---

### Additional information

Requires: https://github.com/ckeditor/ckeditor5-dev/pull/551

### CODE REVIEW

**Comment by pomek** [comment]
image

File: scripts/docs/build-docs.js [hunk 1]
```diff
@@ -27,7 +27,10 @@ function buildDocs() {
 	if ( skipApi ) {
 		promise = Promise.resolve();
 	} else {
-		promise = buildApiDocs();
+		promise = buildApiDocs()
+			.catch( () => {
```
Comment (ma2ciek) [inline scripts/docs/build-docs.js]: We can miss a random error easily with such catching.

Comment (pomek) [reply scripts/docs/build-docs.js]: I didn't find a better way.

Comment (ma2ciek) [reply scripts/docs/build-docs.js]: even the `catch( err => { console.error( err ); process.exitCode = 1; } )` will be better.

**Pull request merged**